Signal/src/contact/OWSContactsManager.m

```diff
@@ -7,6 +7,8 @@
 
 typedef BOOL (^ContactSearchBlock)(id, NSUInteger, BOOL *);
 
+NSString *const OWSContactsManagerContactsDidChangeNotification = @"OWSContactsManagerContactsDidChangeNotification";
```

Comment:
Yeah, maybe disambiguating both ways... something like renaming `Contact` to `SystemContact` and `SignalRecipient` to `SignalContact`